src/components/ListItem.jsx

```diff
@@ -21,6 +21,14 @@ export function ListItem({ name, data, listToken }) {
 		}
 	};
 
+	function handleDeleteItem() {
+		if (window.confirm(`Are you sure you wish to delete ${name}?`)) {
```

Comment:
Potential improvement - currently the user is unable to access the `Cancel` button via keyboard

Reply:
It works from my keyboard?

Reply:
My experience with window.confirm has been that you can use `esc` and `return` to cancel / ok respectively, but you can't use `tab` to navigate between cancel and ok which would be a nice feature.

Reply:
@tinazarb I removed the else statement in handleDeleteItem() which was throwing the error when canceling from the confirm window.  I experimented some with .focus() on the confirm window to allow navigation between the keys with the tab key, but wasn't successful for now.